taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+from string import ascii_uppercase
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    license_number = forms.CharField(required=False)
+
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number",
+        )
+
+    def clean_license_number(self):
+        return validate_license_number(self.cleaned_data.get("license_number"))
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = get_user_model()
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        return validate_license_number(self.cleaned_data.get("license_number"))
+
+
+class CarCreateForm(forms.ModelForm):
```

Comment:
seems like this form can be also used for car update !?

Reply:
If you're talking about updating whole car data, for sure! Idk, it wasn't in the task, I don't like to make much changes to pulled logic, only add my.
You want me to refactor that?